cmd/limactl/start.go

```diff
@@ -56,6 +60,7 @@ $ cat template.yaml | limactl start --name=local -
 	// TODO: "survey" does not support using cygwin terminal on windows yet
 	startCommand.Flags().Bool("tty", isatty.IsTerminal(os.Stdout.Fd()), "enable TUI interactions such as opening an editor, defaults to true when stdout is a terminal")
 	startCommand.Flags().String("name", "", "override the instance name")
+	startCommand.Flags().String("yq", "", "modify the template inplace")
```

Comment:
Maybe:
```suggestion
	startCommand.Flags().String("set", "", "modify the template inplace, using yq syntax")
```

Reply:
- I guess we have to prohibit modifying images?
- Maybe we want to treat this as an experimental?
- Please update `README.md` too

Reply:
Why is it not allowed to modified images, from the templates ?

Reply:
Might be ok for new instances, but not allowable for existing instances, obviously

Reply:
Right. But this new tool is not very different, from doing it yourself in the editor...